vm/message/src/unsigned_message.rs

```diff
@@ -215,4 +221,29 @@ pub mod json {
             params: Serialized::new(base64::decode(&m.params).map_err(de::Error::custom)?),
         })
     }
+
+    pub mod vec {
+        use super::*;
+        use forest_json_utils::GoVecVisitor;
+        use serde::ser::SerializeSeq;
+
+        pub fn serialize<S>(m: &[UnsignedMessage], serializer: S) -> Result<S::Ok, S::Error>
+        where
+            S: Serializer,
+        {
+            let mut seq = serializer.serialize_seq(Some(m.len()))?;
+            for e in m {
+                seq.serialize_element(&UnsignedMessageJsonRef(e))?;
+            }
+            seq.end()
```

Comment:
seems like this block is repeated a few times, is it worth to put in a closure or function?

Reply:
I guess the serialize_element function is different and it maybe generalizing it will make it harder to read?